src/platform/storm/src/main.rs

```diff
@@ -373,7 +373,8 @@ pub unsafe fn reset_handler() {
         drivers::spi::Spi::new(&mut sam4l::spi::SPI),
         84);
     spi.config_buffers(&mut spi_read_buf, &mut spi_write_buf);
-    sam4l::spi::SPI.init(spi as &hil::spi_master::SpiCallback);
+    sam4l::spi::SPI.set_client(spi);
+    sam4l::spi::SPI.init();
```

Comment:
I think this is actually something the client (i.e. `drivers::spi::Spi`) should probably call, since we don't want to enable the clocks, etc, until someone actually wants to use it. Right?